lib/ansible/modules/windows/win_package.py

```diff
@@ -163,14 +169,15 @@
     product_id: '{CF2BEA3C-26EA-32F8-AA9B-331F7E34BA97}'
     arguments: /install /passive /norestart
 
-- name: Install Visual C thingy with list of arguments instead of a string
+- name: Install Visual C thingy with list of arguments instead of a string, and permanent log
   win_package:
     path: http://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x64.exe
     product_id: '{CF2BEA3C-26EA-32F8-AA9B-331F7E34BA97}'
     arguments:
     - /install
     - /passive
     - /norestart
+    log_path: "D:\\logs\\vcredist_x64-exe-{{lookup('pipe', 'date +%Y%m%dT%H%M%S')}}.log"
```

Comment:
It's best to show examples where we don't have to quote, or use single quotes. This could just be

`log_path: D:\logs\vcredist_x64-exe-{{ lookup('pip', 'date +%Y%m%dT%H%M%S') }}.log`

Reply:
right